.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  # Dependencies listed in go.mod
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```

Comment:
Maybe we should add some labels here too?

```yaml
labels:
  - "kind/cleanup"
  - "priority/important-soon"
  - "wg/security-audit"
```

Reply:
Make sense.

Beside these labels any other label to add ?

Reply:
Those were the only ones I saw that seemed like good defaults.

Reply:
ok. 

For "go.mod" adding all these, but for "github-action" only adding "kind/cleanup"